shell/platform/embedder/embedder.h

```diff
@@ -1568,6 +1568,24 @@ typedef enum {
   kFlutterLayerContentTypePlatformView,
 } FlutterLayerContentType;
 
+typedef struct {
+  /// The size of this struct. Must be sizeof(FlutterRegion).
+  size_t struct_size;
+  /// Number of rectangles in the region.
+  size_t num_rects;
```

Comment:
We've been inconsistent, but I believe the preferred naming pattern is `rects_count`